### ISSUE TITLE
There is some kind of problem "RuntimeError: Error(s) in loading state_dict for SdxlUNet2DConditionModel"

### ISSUE THREAD

**Issue by Creaproot** [bmaltais/kohya_ss#2737]
Greetings) I have a learning error. It outputs this (the 3070 TI card, I'm trying to train something, I get the following error message. I have no idea how to fix it ((( But I'm sure everything is easy for a knowledgeable person. I'm asking for help!!

![2024-08-24_03-07-13](https://github.com/user-attachments/assets/7dfa9239-6e33-4725-8b1e-0e2c6d47cb84)

**Comment by b-fission** [comment]
You're trying to use a Lora as your checkpoint? That's not going to work. Use an actual model to train on, like SDXL 1.0

**Comment by Creaproot** [comment]
Indeed, everything is as you said! Thank you very much)) Very helpful)

> Вы пытаетесь использовать Lora в качестве контрольной точки? Это не сработает. Используйте реальную модель для обучения, например SDXL 1.0

Indeed, everything is as you said! Thank you very much)) Very helpful)
